central/config/datastore/datastore_test.go

```diff
@@ -444,7 +444,7 @@ func TestValidateConfigAndPopulateMissingDefaults(t *testing.T) {
 			upsertedConfig: nil,
 		},
 		"Missing private config gets fully configured when Features activated": {
-			enabledFlags: []string{features.UnifiedCVEDeferral.EnvVar()},
+			enabledFlags: []string{features.PlatformComponents.EnvVar()},
 			initialConfig: &storage.Config{
 				PublicConfig:  samplePublicConfig,
```

Comment:
nit: Just add `samplePlatformConfig` to initialConfig instead of having to set an unrelated envVar in this test.